app/src/main/java/com/beemdevelopment/aegis/ui/views/EntryListView.java

```diff
@@ -149,6 +152,23 @@ public long getMillisTillNextRefresh() {
             }
         });
 
+        final int rvInitialPaddingLeft = _recyclerView.getPaddingLeft();
+        final int rvInitialPaddingTop = _recyclerView.getPaddingTop();
+        final int rvInitialPaddingRight = _recyclerView.getPaddingRight();
+        final int rvInitialPaddingBottom = _recyclerView.getPaddingBottom();
+
+        ViewCompat.setOnApplyWindowInsetsListener(_recyclerView, (targetView, windowInsets) -> {
+            Insets insets = windowInsets.getInsets(WindowInsetsCompat.Type.systemBars() | WindowInsetsCompat.Type.displayCutout());
+            // left and right padding seems to be handled by fitsSystemWindows="true" on the CoordinatorLayout in activity_main.xml
+            targetView.setPadding(
+                    rvInitialPaddingLeft,
+                    rvInitialPaddingTop,
+                    rvInitialPaddingRight,
+                    rvInitialPaddingBottom + insets.bottom
+            );
```

Comment:
This appears to cause some inconsistency, horizontal padding is now larger than the vertical padding.

Reply:
I can't spot this issue, the padding looks the same for me, just with extra added at the bottom for the navigation bar. Could you provide some example?

Reply:
Actually I can't. I could have sworn I saw some inconsistency, but I'm not able to reproduce it now. All good then!